documentation/oracle/governance/adding_a_price_identifier.md

```diff
@@ -0,0 +1,48 @@
+# Adding a Price Identifier
+
+## What is a price identifier?
+
+A [price identifier](../../synthetic_tokens/glossary.md) is a natural language descriptor of the reference index determining how much collateral is needed for a token sponsor to be properly collateralized. 
+Because DVM voters need to be able to vote on the value of this price identifier when disputes are raised, the DVM keeps a list of approved price identifiers. 
+
+An example of a price identifier is “GOLD_APR20”. 
+This price identifier would be documented in the UMIP that was approved by the UMA tokenholders. 
+That UMIP would contain more information about how to calculate the price identifier. 
+
+## Who controls the list of approved price identifiers?
+For each deployment of the DVM, the list of approved price identifiers is controlled by the ```IdentifierWhitelist``` contract.
```

Comment:
why does this have three quotes around it? wont this make it a code block? Should it not just be one tick like this:
```suggestion
For each deployment of the DVM, the list of approved price identifiers is controlled by the `IdentifierWhitelist` contract.
```

Reply:
In the preview, it showed up inline, not as a code block, but let me change the ticks to match.